index.js

```diff
@@ -16,37 +16,52 @@ Toolkit.run(async (tools) => {
     console.log("Couldn't find any commits in this event, incrementing patch version...");
   }
 
+  const tagPrefix = process.env['INPUT_TAG-PREFIX'] || '';
   const messages = event.commits ? event.commits.map((commit) => commit.message + '\n' + commit.body) : [];
 
   const commitMessage = process.env['INPUT_COMMIT-MESSAGE'] || 'ci: version bump to {{version}}';
-  console.log('messages:', messages);
-  const commitMessageRegex = new RegExp(commitMessage.replace(/{{version}}/g, 'v\\d+\\.\\d+\\.\\d+'), 'ig');
+  console.log('commit messages:', messages);
+  const commitMessageRegex = new RegExp(commitMessage.replace(/{{version}}/g, `${tagPrefix}\\d+\\.\\d+\\.\\d+`), 'ig');
   const isVersionBump = messages.find((message) => commitMessageRegex.test(message)) !== undefined;
 
   if (isVersionBump) {
-    tools.exit.success('No action necessary!');
+    tools.exit.success('No action necessary because we found a previous bump!');
     return;
   }
 
+  // input wordings for MAJOR, MINOR, PATCH, PRE-RELEASE
   const majorWords = process.env['INPUT_MAJOR-WORDING'].split(',');
   const minorWords = process.env['INPUT_MINOR-WORDING'].split(',');
+  // patch is by default empty, and '' would always be true in the includes(''), thats why we handle it separately
+  const patchWords = process.env['INPUT_PATCH-WORDING'] ? process.env['INPUT_PATCH-WORDING'].split(',') : null;
```

Comment:
@cmario92 this was a missing piece to not break existing default settings with patch.

Reply:
can you test this with your config please? then we can merge

Reply:
I tested both with patch wording and generic wording commit messages to trigger the prc in my case.

all working as expected!